Sources/MongoMobile/MongoMobile.swift

```diff
@@ -75,25 +75,63 @@ private func getStatusExplanation(_ status: OpaquePointer?) -> String {
 
 /// A class containing static methods for working with MongoMobile.
 public class MongoMobile {
-    private static var libraryInstance: OpaquePointer?
+    public static let shared = MongoMobile()
```

Comment:
from a user perspective it seems annoying to have to add `.shared` every time. is there a reason for doing it this way rather than just internally managing the shared instance and passing through static method calls to it?
at the very least we should add a docstring about what this is

Reply:
This is the idiomatic way to do singletons in Swift. It seems generally popular in the community, and is what `UIKit` and `Foundation` both use for their singletons

Reply:
if it's consistent with other libraries then that sounds fine, let's just add docstring then.